magento/pwa-studio
# fix(loading-regions): state loading stuck when "US" is not in the supported countries

## Description

I've added an environment variable that fallbacks to **"US"** and replace all occurence of the latter country by `process.env.DEFAULT_COUNTRY_CODE`.  
This patch will allow the solution developer to specify the default selected country as he likes. If it's not specified it will fallback to **"US"**.  
Please note that  **Default Country** is an option in the magento store configuration but unfortunately it's not return neither bu the graphql nor by the rest api. 

## Related Issue





Closes #3059.

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to the .env file in venia-concept and uncomment `DEFAULT_COUNTRY_CODE`
2. Write down your preferred default country
3. Add anything to the cart and go to checkout page while guest to see your preferred country selected by default.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
